pennylane_lightning/src/tests/Test_AdjDiff.cpp

```diff
@@ -11,13 +11,12 @@
 #include <catch2/catch.hpp>
 
 #include "AdjointDiff.hpp"
-#include "StateVectorManaged.hpp"
-#include "StateVectorRaw.hpp"
+#include "StateVectorRawCPU.hpp"
 #include "Util.hpp"
 
 #include "TestHelpers.hpp"
 
-#ifndef _USE_MATH_DEFINES
+#if !defined(_USE_MATH_DEFINES)
```

Comment:
Is there any difference in how these two preprocessor directives are compiled?

Reply:
I don't remember why I changed this... Both work exactly the same, but using `defined()` is better some cases as one cand add additional conditions.